imports/plugins/included/product-variant/components/variantForm.js

```diff
@@ -536,39 +642,43 @@ class VariantForm extends Component {
           />
         </Components.SettingsCard>
 
-        <Components.SettingsCard
-          enabled={this.state.inventoryManagement}
-          expandable={true}
-          i18nKeyTitle="productVariant.inventoryManagement"
-          name="inventoryManagement"
-          packageName={"reaction-product-variant"}
-          saveOpenStateToPreferences={true}
-          showSwitch={true}
-          title="Inventory Tracking"
-          onSwitchChange={this.handleCheckboxChange}
-        >
-          <div className="row">
-            {this.renderQuantityField()}
-            <div className="col-sm-6">
-              <Components.TextField
-                i18nKeyLabel="productVariant.lowInventoryWarningThreshold"
-                i18nKeyPlaceholder="0"
-                placeholder="0"
-                label="Warn At"
-                name="lowInventoryWarningThreshold"
-                ref="lowInventoryWarningThresholdInput"
-                value={this.variant.lowInventoryWarningThreshold}
-                onBlur={this.handleFieldBlur}
-                onChange={this.handleFieldChange}
-                onReturnKeyDown={this.handleFieldBlur}
-                validation={this.props.validation}
-              />
+        {
+          !this.props.currentProduct.isDigital &&
+
+          <Components.SettingsCard
+            enabled={this.state.inventoryManagement}
+            expandable={true}
+            i18nKeyTitle="productVariant.inventoryManagement"
+            name="inventoryManagement"
+            packageName={"reaction-product-variant"}
+            saveOpenStateToPreferences={true}
+            showSwitch={true}
```

Comment:
Acording to preference, `showSwitch={true}` and `expandable={true}` could be `showSwitch` and `expandable`

Reply:
Thanks for this review. I didn't work on this component, but the Eslint settings for the project are not particular on how boolean props should be passed. Also, I do not know how the component works internally, so it would be risky toying with it. Thanks.